### DISCUSSION TITLE
Welcome to Harmless' Lua Script Discussions!

### DISCUSSION THREAD

**Discussion by Harmless05** [Harmless05/harmless-lua#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Suggest new features or Lua scripts

**Comment by xesdoog** [comment]
Hey there @Harmless05 !

I would like to express my gratitude towards your awesome contributions to this community. 
I would also like to have your permission to implement a slightly tweaked version of Shift-Drift into other scripts. I like it so much I can't even play GTA without it lol Thank you for making it!

_BTW, that cat is so $@*# beautiful I wanna squish it_

**Comment by Harmless05** [comment]
Thank you @xesdoog! 
I'm trying my best to make the community better with Lua scripts and other useful scripts/programs.

Feel free to use Shift-Drift, just give me a little credit :D

**Comment by Harmless05** [comment]
If you'd like, you can create a pull request for my current version of Shift-Drift too at [YimMenu-Lua/Harmless-Scripts](https://github.com/YimMenu-Lua/Harmless-Scripts).

I haven't had much time to update my current script due to school and also writing a PowerShell script to automatically update YimMenu.

**Reply by xesdoog** [reply]
Will do
Although I didn't add anything significant other than a check for vehicle type and a script reset in a few cases where it felt necessary. I will revert the name back to the original (_I renamed it SlidyBoiGoesSkrrt. I know it's stupid_ 😓) then open a PR.

I like your ideas like the PowerShell updater for instance. [Deadlineem](https://github.com/Deadlineem) is doing something similar with a batch script. I have one in mind for LUAs that could be a simple template which we could all use for example we upload it once to YimMenu-Lua and then every script author that wants an auto-update feature could simply download it, add a link to their repo inside the updater then upload it with their original script. 
Idk, I'm just thinking out loud and I'm not sure this would even be possible since I don't know if Yim's LUA engine has LuaSocket.

**Reply by Harmless05** [reply]
Yes, I saw it a few days ago and looked quite good. I chose PowerShell because it's easier for me to write scripts in, as I've learnt it more than Batch. I originally planned to have the updater (download/update YimMenu and Yim Luas) in my launcher/injector, but I haven't worked on it for a good while.

YimMenu's lua still needs some work. It lacks quite good/useful features from ImGui. As for updating, Yimura wants to add something directly to YimMenu and can download Luas from there.
